Makefile

```diff
@@ -11,3 +11,9 @@ build-alpine:
 	docker build -f Dockerfile-alpine -t jakzal/phpqa:alpine .
 .PHONY: build-alpine
 
+
+NIGHTLY_TAG := jakzal/phpqa:nightly-$(shell date +%y%m%d)
```

Comment:
Shall we just use `latest` and `alpine`? These are meant for the latest versions of both image flavours.

Reply:
Hm.. this is not how daily builds usually work, so I got confused :) There's usually one (latest) daily build available.

I usually create tags right after new features or fixes are merged, and then a merge triggers a build. 

How would creating a tag each day improve the situation for you? I'm worried we'd be generating lots of tags on docker hub.

Reply:
Here's how:

Reply:
Here's how:

Reply:
Generating lot of tags on docker hub is not a problem, I did mention this in OG post in this PR. But yes, it would be nice to optimize this later to NOT create new tag if nothing changed (no new versions of packages)

Here's how it improves situation for me:

1. New patch release of Easy Coding Standard has been released which solves some issues I have
1. There is no new release of phpqa, but there is new nightly build, so I just require jakzal/phpqa:nightly-currentday
1. New major release of Easy Coding Standard has been releases, which changes configuration format from .neon to .yaml
1. Build of my project is triggered again
1. Nothing is broken because I've locked to nightly-currentday instead of :latest